lib/full-site-editing/edit-site-page.php

```diff
@@ -119,6 +119,7 @@ static function( $classes ) {
 		'styles'                               => gutenberg_get_editor_styles(),
 		'defaultTemplateTypes'                 => $indexed_template_types,
 		'defaultTemplatePartAreas'             => get_allowed_block_template_part_areas(),
+		'__unstableHomeTemplate'               => gutenberg_resolve_home_template(),
```

Comment:
You can comment out this line to test the warning when the method cannot resolve the template.